api-breakages.txt

```diff
@@ -18,3 +18,4 @@ API breakage: func Configuration.MultilineStringReflowBehavior.hash(into:) has b
 API breakage: func Configuration.MultilineStringReflowBehavior.encode(to:) has been removed
 API breakage: var Configuration.MultilineStringReflowBehavior.hashValue has been removed
 API breakage: constructor Configuration.MultilineStringReflowBehavior.init(from:) has been removed
+API breakage: enum LineType.implementationOnlyImport has been added as a new enum case
```

Comment:
This check is still failing; the message needs to be copied verbatim from the error in the CI log:

`API breakage: enumelement LineType.implementationOnlyImport has been added as a new enum case`

(`enumelement`, not `enum`)